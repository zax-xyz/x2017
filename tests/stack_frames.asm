FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK B VAL 250
    PRINT STK A
    PRINT STK B
    CAL VAL 1
    PRINT STK A
    PRINT STK B
    RET
FUNC LABEL 1
    MOV STK A VAL 151
    MOV STK B VAL 23
    PRINT STK A
    PRINT STK B
    RET
