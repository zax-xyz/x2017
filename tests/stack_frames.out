5
250
151
23
5
250
